pkg/promotion/promotion.go

```diff
@@ -130,12 +130,13 @@ func (o *Options) Bind(fs *flag.FlagSet) {
 	fs.StringVar(&o.Repo, "repo", "", "Limit repos affected to this repo.")
 }
 
+var masterBranches = regexp.MustCompile(`^master.*$`)
 var threeXBranches = regexp.MustCompile(`^(release|enterprise|openshift)-3\.[0-9]+$`)
 var fourXBranches = regexp.MustCompile(`^(release|enterprise|openshift)-(4\.[0-9]+)$`)
 
 func FlavorForBranch(branch string) string {
 	var flavor string
-	if branch == "master" {
+	if masterBranches.MatchString(branch) {
```

Comment:
The glob he have today is [here](https://github.com/openshift/release/blob/e360310226b0d28327e779acc95c2fb6748beb43/core-services/prow/02_config/_plugins.yaml#L1431-L1434):

```yaml
    ci-operator/config/**/*master*.yaml:
      name: ci-operator-master-configs
      additional_namespaces:
      - ci-stg
```

That *would* match `not-really-master-branch`